pkg/app/piped/sourcedecrypter/decrypter.go

```diff
@@ -49,14 +50,12 @@ func DecryptSecrets(appDir string, enc config.SecretEncryption, dcr secretDecryp
 
 	for _, t := range enc.DecryptionTargets {
 		targetPath := filepath.Join(appDir, t)
-		tmpl, err := template.ParseFiles(targetPath)
+		fileName := filepath.Base(targetPath)
+		tmpl, err := template.New(fileName).Funcs(sprig.TxtFuncMap()).Option("missingkey=error").ParseFiles(targetPath)
```

Comment:
How about like this?
```suggestion
		tmpl := template.
			New(fileName).
			Funcs(sprig.TxtFuncMap()).
			Option("missingkey=error")
		tmpl, err := tmpl.ParseFiles(targetPath)
		if err != nil {
			return fmt.Errorf("failed to parse decryption target %s (%w)", t, err)
		}
```